src/libData/AccountData/Account.cpp

```diff
@@ -301,6 +279,25 @@ vector<h256> Account::GetStorageKeyHashes(bool temp) const {
       m_address, temp);
 }
 
+void Account::GetUpdatedStates(
+    std::map<std::string, bytes>& t_states,
+    std::vector<std::string>& toDeleteIndices) const {
+  ContractStorage2::GetContractStorage().FetchUpdatedStateValuesForAddress(
+      GetAddress(), t_states, toDeleteIndices);
+}
+
+void Account::UpdateStates(const Address& addr,
+                           const std::map<std::string, bytes>& t_states,
+                           const std::vector<std::string>& toDeleteIndices,
+                           bool temp, bool revertible) {
+  ContractStorage2::GetContractStorage().UpdateStateDatasAndToDeletes(
+      addr, t_states, toDeleteIndices, m_storageRoot, temp, revertible);
+  if (m_address == Address()) {
```

Comment:
Can just use if (m_address), also other place can change like this.

Reply:
`if (m_address)` means `m_address == Address()` or `!= Address()`. Decided to remain in sake of clarification.

Reply:
m_address == Address() is not efficient, it will construct a new object and compare and discard it. if (m_address) means address is not empty, here need to use if (!m_address)